src/applications/disability-benefits/all-claims/tests/containers/WizardContainer.unit.spec.jsx

```diff
@@ -5,29 +5,32 @@ import { shallow } from 'enzyme';
 import WizardContainer from '../../containers/WizardContainer';
 import { WIZARD_STATUS_COMPLETE } from 'platform/site-wide/wizard';
 
+import { WIZARD_STATUS } from '../../constants';
+
 describe('Wizard Container', () => {
-  global.status = '';
-  const props = {
-    setWizardStatus: value => {
-      global.status = value;
-    },
-  };
+  beforeEach(() => {
+    sessionStorage.removeItem(WIZARD_STATUS);
+  });
+  after(() => {
+    sessionStorage.removeItem(WIZARD_STATUS);
+  });
```

Comment:
would it make more sense to put this all in an afterEach? Is there concern that sessionStorage will contain the Wizard Status from a test outside this file?

Reply:
I didn't have a concern about sessionStorage outside of this test, but I figured it'd be safer to cleanup. Would you recommend using `afterEach` then?

Reply:
Typically it's best to establish the pattern of cleaning up in an `afterEach`. However, I understand wanting to be sure that your environment is clean before your tests are run, so I'll leave it up to you. If you want to keep it as is, that's fine.